package.json

```diff
@@ -806,10 +813,22 @@
         "description": "Restarts a composition of containers",
         "category": "Docker"
       },
+      {
+        "command": "vscode-docker.compose.start",
+        "title": "Compose Restart",
+        "description": "Builds and starts a composition of containers",
+        "category": "Docker"
+      },
+      {
+        "command": "vscode-docker.compose.stop",
+        "title": "Compose Restart",
+        "description": "Stops a composition of containers (non-destructive)",
```

Comment:
Stops a composition of containers without removing them.

(see https://docs.docker.com/compose/reference/stop/)